### PR TITLE
Create module layout for new components

### PR DESCRIPTION
**Pull request by pocmo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,16 @@ We encourage you to participate in this open source project. We love Pull Reques
 
 High-level components for building browser(-like) apps.
 
+* **Engine** - An abstract layer on hiding the actual browser engine implementation.
```
Comment (csadilek) [inline README.md]: nit: remove 'on' or for hiding? to hide?

File: components/browser/session/build.gradle [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+dependencies {
+    compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+}
+
+apply from: '../../../publish.gradle'
+ext.configurePublish(
+        'org.mozilla.components',
+        'engine',
```
Comment (csadilek) [inline components/browser/session/build.gradle]: This should be session?

File: components/browser/toolbar/src/main/java/mozilla/components/session/BrowserToolbar.kt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.session
```
Comment (csadilek) [inline components/browser/toolbar/src/main/java/mozilla/components/session/BrowserToolbar.kt]: Shoud this be mozilla/components/toolbar?

File: README.md (hunk 1 above)
Comment (pocmo) [reply README.md]: Oops. Yeah, I rewrote that sentence a couple of times 😅

**Comment by pocmo** [comment]
@csadilek Okay, pushed new version addressing all your review comments. :)

**Pull request merged**